MHW_ModManager/MainForm.zh.resx

```diff
@@ -0,0 +1,1981 @@
+        AP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/FhYW/8vLy///////7+/v9Dg4ONABAQH3BAQE/wMD
+        A/8BAQH/AgIC/wICAv8wMDD35+fn/P/////Ly8v/FhYW/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAA
+        AP8AAAD/gYGB///////////+////dP///wD///8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA////AP//
+        /wn///+q//////Pz8/9NTU3/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/Pz8//+/v
+        7///////////uP///w3///8A////AP///w3///+4/////+/v7/8/Pz//AAAA/wAAAP8AAAD/AAAA/wAA
+        AP8AAAD/AAAA/wAAAP9KSkr/8vLy//////////+s////Cv///wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAD///8A////AP///3H////9/////4WFhf8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAA
+        AP8WFhb/y8vL///////////ko6OjRgAAAHgEBATiBAQE/wMDA/8EBAT+AAAAzVlZWYH6+vrp/////8vL
+        y/8WFhb/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP+BgYH///////////7///90////AP//
+        /wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD///8A////Cf///6r/////8/Pz/01NTf8AAAD/AAAA/wAA
+        AP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8/Pz//7+/v//////////+4////Df///wD///8A////Df//
+        /7j/////7+/v/z8/P/8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/0pKSv/y8vL/////////
+        /6z///8K////AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD///8A////cf////3/////hYWF/wAA
+        AP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/xYWFv/Ly8v//////////+T+/v4tAAAAAwIC
+        AkgDAwPWAwMD/wMDA8UAAAAx+/v7Lv///+T/////y8vL/xYWFv8AAAD/AAAA/wAAAP8AAAD/AAAA/wAA
+        AP8AAAD/AAAA/4GBgf///////////v///3T///8A////AAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAP//
+        /wD///8J////qv/////z8/P/TU1N/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/z8/
+        P//v7+///////////7j///8N////AP///wD///8N////uP/////v7+//Pz8//wAAAP8AAAD/AAAA/wAA
+        AP8AAAD/AAAA/wAAAP8AAAD/SkpK//Ly8v//////////rP///wr///8AAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAA////AP///wD9/f1y/////f////+FhYX/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAA
+        AP8AAAD/FhYW/8vLy///////////5P///y3FxcUAAAAACQICAqADAwP/AgICiAAAAAH+/v4t////5P//
+        ///Ly8v/FhYW/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/gYGB///////////+////dP//
+        /wD///8AAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA////AP///wn///+q//////Pz8/9NTU3/AAAA/wAA
+        AP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/Pz8//+/v7///////////uP///w3///8A////AP//
+        /w3///+4/////+/v7/8/Pz//AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP9KSkr/8vLy////
+        //////+s////Cv///wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
This will not work. general text strings should never be added to form resource files. They will be removed by visual studio whenever something in the designer changes. You will need to use a separate resource file.

Reply:
Basically any non UI string needs to go into a separate resource file that you get the strings in code from. The mainform.resx is auto-generated from the visual studio form designer. So you can modify the UI for whatever language using the designer and that goes into the mainform.resx for whatever language i think.

Reply:
I think it is possible to change the multilingual setup method without using vs's multilingual function. Using external JSON files to transfer multiple languages is more variable, using attribute names as indexes. I will finish this part as soon as possible.